types/natsu.d.ts

```diff
@@ -2,3 +2,15 @@ export type NatsPortReq<T> = {
   headers: any;
   body: T;
 };
+
+export interface NatsInfo {
```

Comment:
NatsInfo is not a good naming, we'll understand that as information for Nats (what nats, server, client, package etc)

NatsService or NatsRPC would be a better naming as it reflects the intention